ui/chat/src/discussion.ts

```diff
@@ -82,6 +85,19 @@ function renderInput(ctrl: ChatCtrl): VNode | undefined {
   });
 }
 
+function prependChatInput(prefix: string): void {
+  const chatInput = document.querySelector('input.mchat__say') as HTMLInputElement;
```

Comment:
page-wide selectors introduces a new asumption that there's only one chat on the page

Reply:
Updated to search within the parent `.mchat` element.